third_party/silabs/BUILD.gn

```diff
@@ -28,8 +34,14 @@ declare_args() {
 
 assert(efr32_sdk_target != "", "efr32_sdk_target must be specified")
 
-group("efr32_sdk") {
-  public_deps = [ efr32_sdk_target ]
+if (silabs_board == "BRD4325A") {  # CCP board
+  group("efr32_sdk") {
+    public_deps = [ efr32_sdk_target ]  # TODO for Si917
```

Comment:
I don't see there difference. Why this If to define the same group/public_deps

Reply:
fixed in the latest commit